virtio-devices/src/transport/pci_device.rs

```diff
@@ -882,11 +882,20 @@ impl PciDevice for VirtioPciDevice {
         let device = device_clone.lock().unwrap();
 
         let mut settings_bar_addr = None;
+        let mut use_64bit_bar = self.use_64bit_bar;
         if let Some(resources) = &resources {
             for resource in resources {
-                if let Resource::PciBar { index, base, .. } = resource {
+                if let Resource::PciBar {
+                    index, base, type_, ..
+                } = resource
+                {
```

Comment:
nit: Did rustfmt put this `{` on a new line, it looks weird... originally it was on the same line

Reply:
Yes it was `rustfmt`, not me :)